src/locale/locales/nl/messages.po

```diff
@@ -0,0 +1,7903 @@
+msgid "Climate"
+msgstr "Klimaat"
+
+#: src/components/dms/ChatEmptyPill.tsx:39
+msgid "Clip 🐴 clop 🐴"
+msgstr "Clip 🐴 clop 🐴"
+
+#: src/components/dialogs/GifSelect.tsx:282
+#: src/components/dialogs/VerifyEmailDialog.tsx:246
+#: src/components/dms/dialogs/SearchablePeopleList.tsx:263
+#: src/components/NewskieDialog.tsx:146
+#: src/components/NewskieDialog.tsx:153
+#: src/components/StarterPack/Wizard/WizardEditListDialog.tsx:117
+#: src/components/StarterPack/Wizard/WizardEditListDialog.tsx:123
+#: src/view/com/modals/ChangePassword.tsx:268
+#: src/view/com/modals/ChangePassword.tsx:271
+#: src/view/com/util/post-embeds/GifEmbed.tsx:197
+msgid "Close"
+msgstr "Sluiten"
+
+#: src/components/dialogs/GifSelect.tsx:169
+msgid "Close GIF dialog"
+msgstr "Sluit GIF-dialoogvenster"
+
+#: src/components/Dialog/index.web.tsx:110
+#: src/components/Dialog/index.web.tsx:256
+msgid "Close active dialog"
+msgstr "Sluit actief dialoogvenster"
+
+#: src/screens/Login/PasswordUpdatedForm.tsx:32
+msgid "Close alert"
+msgstr "Sluit waarschuwing"
+
+#: src/view/com/util/BottomSheetCustomBackdrop.tsx:36
+msgid "Close bottom drawer"
+msgstr "Sluit onderste lade"
+
+#: src/components/dialogs/GifSelect.tsx:276
+msgid "Close dialog"
+msgstr "Sluit dialoogvenster"
+
+#: src/view/com/lightbox/ImageViewing/components/ImageDefaultHeader.tsx:36
+msgid "Close image"
+msgstr "Sluit afbeelding"
+
+#: src/view/com/lightbox/Lightbox.web.tsx:129
+msgid "Close image viewer"
+msgstr "Sluit afbeeldingsviewer"
+
+#: src/view/shell/index.web.tsx:67
+msgid "Close navigation footer"
+msgstr "Sluit navigatievoettekst"
+
+#: src/components/Menu/index.tsx:229
+#: src/components/TagMenu/index.tsx:261
+msgid "Close this dialog"
+msgstr "Sluit dit dialoogvenster"
+
+#: src/view/shell/index.web.tsx:68
+msgid "Closes bottom navigation bar"
+msgstr "Sluit onderste navigatiebalk"
+
+#: src/screens/Login/PasswordUpdatedForm.tsx:33
+msgid "Closes password update alert"
+msgstr "Sluit waarschuwing voor bijwerken wachtwoord"
+
+#: src/view/com/composer/Composer.tsx:552
+#: src/view/com/lightbox/ImageViewing/components/ImageDefaultHeader.tsx:37
+msgid "Closes viewer for header image"
+msgstr "Sluit viewer voor headerafbeelding"
+
+#: src/view/com/notifications/FeedItem.tsx:265
+msgid "Collapse list of users"
+msgstr "Dichtvouwen gebruikerslijst"
+
+#: src/view/com/notifications/FeedItem.tsx:470
+msgid "Collapses list of users for a given notification"
+msgstr "Dichtvouwen gebruikerslijst voor een bepaalde melding"
+
+#: src/screens/Settings/AppearanceSettings.tsx:97
+msgid "Color mode"
+msgstr "Kleurmodus"
+
+#: src/screens/Onboarding/index.tsx:38
+#: src/screens/Onboarding/state.ts:84
+msgid "Comedy"
+msgstr "Komedie"
+
+#: src/screens/Onboarding/index.tsx:24
+#: src/screens/Onboarding/state.ts:85
+msgid "Comics"
+msgstr "Stripboeken"
+
+#: src/Navigation.tsx:275
+#: src/view/screens/CommunityGuidelines.tsx:34
+msgid "Community Guidelines"
+msgstr "Communityrichtlijnen"
```

Comment:
I think this is fine, but a complete translation would be

```suggestion
msgstr "Gemeenschapsrichtlijnen"
```

Reply:
Registered as translateable term. Applied changes everywhere.